### PR TITLE
Onboarding branch pull request

### PR DESCRIPTION
**Pull request by bdowds**
First draft of 'Banking App'.

### CODE REVIEW

File: OnboardingExperience/User.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace OnboardingExperience
+{
+    class User
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string UserName { get; set; }
+        public int Pin { get; set; }
+    }
```
Comment (codytwinton) [inline OnboardingExperience/User.cs]: Missing `public bool IsAccountOwner { get; set; }`

File: OnboardingExperience/Program.cs [hunk 2]
```diff
@@ -1,14 +1,118 @@
...
+                if (answer.ToLower() == "y") { return true; }
+                else if (answer.ToLower() == "n") { return false; }
+                else { Console.WriteLine("Please enter either a 'y' for Yes, or a 'n' for No."); }
+            } while (true);
+        }
+
+
+
+        //Based on the character passed through, Returns either the string entered by the user or a pin number.
+        private static object AskQuestion(string question, char returnType)
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: Clever, but not type safe. I would prefer multiple, type safe methods to a single, all purpose method. Even were you to use this, you should create a `DataType` enum over a `char` that could be anything.

File: OnboardingExperience/Program.cs [hunk 3]
```diff
@@ -1,14 +1,118 @@
...
+                Console.Clear();
+
+            } while (!isConfirmed);
+
+            Console.WriteLine("\nAccount has been created successfully!");
+            Console.WriteLine("\nPlease Login");
+
+            //Asks the user to Login with a Username and Pin
+            //Loops until the correct Username and Pin are entered
+            do
```
Comment (codytwinton) [inline OnboardingExperience/Program.cs]: There is a lot going on in this Main method that should probably be abstracted into some more methods.

**Comment by bdowds** [comment]
Transferred most of Main code into Methods of the User class.
Added an 'IsAccountOwner' variable.

**Pull request merged**